ui/components/app/edit-gas-fee-popover/network-statistics/status-slider/status-slider.js

```diff
@@ -16,55 +17,67 @@ const GRADIENT_COLORS = [
   '#C54055',
 ];
 
-const STATUS_INFO = {
-  low: {
-    statusLabel: 'notBusy',
-    tooltipLabel: 'lowLowercase',
-    color: GRADIENT_COLORS[0],
-  },
-  stable: {
-    statusLabel: 'stable',
-    tooltipLabel: 'stableLowercase',
-    color: GRADIENT_COLORS[4],
-  },
-  high: {
-    statusLabel: 'busy',
-    tooltipLabel: 'highLowercase',
-    color: GRADIENT_COLORS[9],
-  },
-};
+const determineStatusInfo = (givenNetworkCongestion) => {
+  const networkCongestion = givenNetworkCongestion ?? 0.5;
+  const colorIndex = Math.round(networkCongestion * 10);
+  const color = GRADIENT_COLORS[colorIndex];
+  const sliderTickValue = colorIndex * 10;
 
-const getStatusInfo = (status) => {
-  if (status <= 0.33) {
-    return STATUS_INFO.low;
-  } else if (status > 0.66) {
-    return STATUS_INFO.high;
+  if (networkCongestion <= 0.33) {
+    return {
+      statusLabel: 'notBusy',
+      tooltipLabel: 'lowLowercase',
+      color,
+      sliderTickValue,
+    };
+  } else if (networkCongestion > 0.66) {
+    return {
+      statusLabel: 'busy',
+      tooltipLabel: 'highLowercase',
+      color,
+      sliderTickValue,
+    };
   }
-  return STATUS_INFO.stable;
+  return {
+    statusLabel: 'stable',
+    tooltipLabel: 'stableLowercase',
+    color,
+    sliderTickValue,
+  };
 };
 
 const StatusSlider = () => {
-  const statusValue = 0.5;
-  const sliderValueNumeric = Math.round(statusValue * 10);
-
-  const statusInfo = getStatusInfo(statusValue);
+  const { gasFeeEstimates } = useGasFeeContext();
+  const statusInfo = determineStatusInfo(gasFeeEstimates.networkCongestion);
 
   return (
-    <NetworkStabilityTooltip statusInfo={statusInfo}>
+    <NetworkStabilityTooltip
+      color={statusInfo.color}
+      tooltipLabel={statusInfo.tooltipLabel}
+    >
       <div className="status-slider">
-        <div className="status-slider__arrow-border">
+        <div className="status-slider__arrow-container">
```

Comment:
I had to add yet another div around the arrow because it didn't seem like it was positioned correctly when `networkCongestion` was 0. We do that by applying a negative margin to the container. We can't apply a negative margin to the border because it already has a margin-left, so we need another element.